ArduPlane/servos.cpp

```diff
@@ -255,29 +255,20 @@ void Plane::channel_function_mixer(SRV_Channel::Aux_servo_function_t func1_in, S
  */
 void Plane::flaperon_update(int8_t flap_percent)
 {
-    if (!SRV_Channels::function_assigned(SRV_Channel::k_flaperon1) ||
-        !SRV_Channels::function_assigned(SRV_Channel::k_flaperon2)) {
+    if (!SRV_Channels::function_assigned(SRV_Channel::k_flaperon_left) &&
+        !SRV_Channels::function_assigned(SRV_Channel::k_flaperon_right)) {
```

Comment:
no, both are correct, just different level of optimisation. Previously we only generated flaperons if both outputs were defined. Now we allow you to have just one defined.